plugins/woocommerce/src/Internal/Admin/Settings/PaymentsProviders/PaymentGateway.php

```diff
@@ -275,8 +275,10 @@ public function get_supports_list( WC_Payment_Gateway $payment_gateway ): array
 	 */
 	public function get_provider_links( WC_Payment_Gateway $payment_gateway, string $country_code = '' ): array {
 		$country_code = strtoupper( sanitize_text_field( $country_code ) );
-		// Validate the country code format - expect ISO 3166-1 alpha-2.
-		if ( strlen( $country_code ) !== 2 || ! ctype_upper( $country_code ) ) {
+		// Validate the country code format, if provided - expect ISO 3166-1 alpha-2.
```

Comment:
Removed unnecessary logging when the country code is sent as an empty string (which is valid).